### PR TITLE
fixes ConfigCenter api, reduce method.

### PR DESCRIPTION
**Pull request by dongzl**
fixes ConfigCenter api, reduce method.

Changes proposed in this pull request:
- remove getDirectly(), isExisted(), update(), persistEphemeral() methods.

### CODE REVIEW

File: sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-api/src/main/java/org/apache/shardingsphere/orchestration/config/api/ConfigCenter.java [hunk 1]
```diff
@@ -26,6 +26,7 @@
  * Config center.
  * 
  * @author wangguangyuan
+ * @author dongzonglei
```
Comment (lovepoem) [inline sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-api/src/main/java/org/apache/shardingsphere/orchestration/config/api/ConfigCenter.java]: When you plan to graduate ,  you will remove all the  author  info of this repo .

Comment (dongzl) [reply sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-api/src/main/java/org/apache/shardingsphere/orchestration/config/api/ConfigCenter.java]: Thank you for your reminder, I will remove it.

Comment (terrymanu) [reply sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-api/src/main/java/org/apache/shardingsphere/orchestration/config/api/ConfigCenter.java]: We can start a discuss in mailing list before graduate, and remove the author in one pull request.

Comment (dongzl) [reply sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-api/src/main/java/org/apache/shardingsphere/orchestration/config/api/ConfigCenter.java]: OK.

**Pull request merged**